src/main/js/main_html5.js

```diff
@@ -86,7 +86,16 @@ require("../../../html5-common/js/utils/environment.js");
       var video = $("<video>");
       video.attr("class", "video");
       video.attr("id", domId);
-      video.attr("preload", "none");
+
+      // [PBW-5470] On Safari, when preload is set to 'none' and the user switches to a
+      // different tab while the video is about to auto play, the browser stops playback but
+      // doesn't fire a 'pause' event, which causes the player to get stuck in 'buffering' state.
+      // Setting preload to 'metadata' (or 'auto') allows Safari to auto resume when the tab is refocused.
```

Comment:
Oh ok, I see. In the comments what I meant is that both `metadata` and `auto` fix the issue.   The difference is that `metadata` only preloads info such as `duration`, etc., but not the video itself. `auto` allows the browser to start buffering the actual media file even before playback starts. I think `metadata` would be preferred in this case.